### PR TITLE
WIP: Link inputs for multiqc

### PR DESCRIPTION
**Pull request by jasonwalker80**
@tmooney I'm testing this now. 

Unsure that's really going to work with Cromwell file localization. Other suggestions?

My goal is to avoid adding a `ln -s` for each BAM/CRAM and it's index to every step of the workflow.

### CODE REVIEW

File: tasks/Qc.wdl [hunk 1]
```diff
@@ -53,6 +80,7 @@ task CollectAggregationMetrics {
   input {
     File input_bam
     File input_bam_index
+    Boolean is_bam
```
Comment (jasonwalker80) [inline tasks/Qc.wdl]: NOTE: @tmooney this was a working example of using `ln -s` in a single step. This was tested and of course worked as expected.

File: tasks/Qc.wdl [hunk 2]
```diff
@@ -14,6 +14,33 @@ version 1.0
...
+  input {
+    File input_bam
+    String base_name
+    Boolean is_bam
+    Int preemptible_tries
+  }
+
+  Int disk_size = ceil(size(input_bam, "GiB")) + 20  
+
+  String bam_link = base_name + "." + if is_bam then "bam" else "cram"
```
Comment (tmooney) [inline tasks/Qc.wdl]: If you don't want to have an `is_bam` variable, you could maybe do something like this in the script:

```bash
INPUT_BAM="~{input_bam}"
EXTENSION="${INPUT_BAM##*.}"
ln -s "~{input_bam}" "~{base_name}.$EXTENSION"
```

Comment (jasonwalker80) [reply tasks/Qc.wdl]: Is there a way I could do this outside of the command{} scope. Would this work there? I will need to define (or resolve) the `bam_link` so I can return it as an output of the Index task.

**Comment by jasonwalker80** [comment]
I am closing with preference for the approach in #46

File: tasks/Qc.wdl (hunk 2 above)
Comment (jasonwalker80) [reply tasks/Qc.wdl]: The answer was no. only command{} is interpreted by the shell.

**Pull request closed without merging**